src/test/resources/db/central-server/pre-populate-separate-central-server.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO inn_reach_recall_user (id, user_id) VALUES ('802f2b63-2c2f-4b75-9d6b-382ff1dfdda3', 'f75ffab1-2e2f-43be-b159-3031e2cfc458');
+
+INSERT INTO central_server(id, name, description, local_server_code, central_server_code, central_server_address, loan_type_id,
+ inn_reach_recall_user_id) VALUES ('cfae4887-f8fb-4e4c-a5cc-34f74e353cf8', 'other', 'other description', 'def12', 'd5ir',
+ 'https://rssandbox-api.iii.com', 'be248906-34e7-4b6e-a09b-a2d31025ae09', '802f2b63-2c2f-4b75-9d6b-382ff1dfdda3');
```

Comment:
`pre-populate-central-server-with-recall-user.sql` and `pre-populate-inn-reach-recall-user.sql` are what you need

Reply:
I spent much time bu couldn't resolve

Reply:
I added 1 file

Reply:
The question is why you need a new file as I pointed existing ones that can be used for this test

Reply:
I cound't adjust existing file